pkg/rpcmiddleware/rpclog/client_interceptors.go

```diff
@@ -33,6 +33,9 @@ func UnaryClientInterceptor(ctx context.Context, opts ...Option) grpc.UnaryClien
 
 		startTime := time.Now()
 		err := invoker(newCtx, method, req, reply, cc, opts...)
+		if shouldSuppressError(err) {
+			return err
+		}
```

Comment:
I don't think we should suppress rate limit errors on the client side, because clients will most likely need to take action (slow down), so they shouldn't ignore these.

Reply:
The problem here is that this is used by the gRPC gateway. If we do not do this, then we get one log line per error.

Reply:
Removed this, let's keep an eye out and return if it becomes an issue.